packages/docusaurus-theme-classic/codeTranslations/ko.json

```diff
@@ -48,16 +48,16 @@
   "theme.docs.sidebar.expandButtonAriaLabel": "사이드바 열기",
   "theme.docs.sidebar.expandButtonTitle": "사이드바 열기",
   "theme.docs.tagDocListPageTitle": "{nDocsTagged} with \"{tagName}\"",
```

Comment:
This is for this page: https://docusaurus.io/tests/docs/tags/a something like "2 docs tagged with "a"". "2 docs tagged" is already translated below so you would have to consider how to splice that with "with "a"" while making it make sense.

Reply:
Gotcha, added.